elements/manganese.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: manganese
```

Comment:
Not sure if it's necessary or not but the title of the element may need to be capitalized as it is on the assignment page.